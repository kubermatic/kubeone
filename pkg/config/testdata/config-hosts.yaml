name: demo
versions:
  kubernetes: '1.14.1'
provider:
  name: 'aws'
hosts:
  - public_address: '1.1.1.1'
    private_address: '1.1.1.2'
    ssh_port: 22
    ssh_username: ubuntu
    ssh_private_key_file: '/home/me/.ssh/id_rsa'
    sshAgentSocket: 'env:SSH_AUTH_SOCK'
  - public_address: '1.1.1.3'
    private_address: '1.1.1.4'
    ssh_port: 22
    ssh_username: ubuntu
    ssh_private_key_file: '/home/me/.ssh/id_rsa'
    sshAgentSocket: 'env:SSH_AUTH_SOCK'
  - public_address: '1.1.1.5'
    private_address: '1.1.1.6'
    ssh_port: 2222
    ssh_username: ubuntu
    ssh_private_key_file: '/home/me/.ssh/id_rsa'
    sshAgentSocket: 'env:SSH_AUTH_SOCK'
