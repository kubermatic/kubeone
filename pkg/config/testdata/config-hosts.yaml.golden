name: demo
versions:
  kubernetes: 1.14.1
hosts:
- sshAgentSocket: env:SSH_AUTH_SOCK
  publicAddress: 1.1.1.1
  privateAddress: 1.1.1.2
  sshPort: 22
  sshUsername: ubuntu
  sshPrivateKeyFile: /home/me/.ssh/id_rsa
- sshAgentSocket: env:SSH_AUTH_SOCK
  publicAddress: 1.1.1.3
  privateAddress: 1.1.1.4
  sshPort: 22
  sshUsername: ubuntu
  sshPrivateKeyFile: /home/me/.ssh/id_rsa
- sshAgentSocket: env:SSH_AUTH_SOCK
  publicAddress: 1.1.1.5
  privateAddress: 1.1.1.6
  sshPort: 2222
  sshUsername: ubuntu
  sshPrivateKeyFile: /home/me/.ssh/id_rsa
apiVersion: kubeone.io/v1alpha1
kind: KubeOneCluster
cloudProvider:
  name: aws
